src/wast-lexer.cc

```diff
@@ -96,10 +96,8 @@ namespace wabt {
 
 struct WastLexer::LexToken {
   Location loc_;
```

Comment:
PR #535 is an attempt to add the default contrustor, may not be correct but it is passing tests.